controller/place.js

```diff
@@ -1,44 +1,82 @@
-var service = { mget: require('../service/mget') };
-var logger = require('pelias-logger').get('api');
+'use strict';
 
-function setup( config, backend ){
+const _ = require('lodash');
+const retry = require('retry');
 
-  // allow overriding of dependencies
-  backend = backend || require('../src/backend');
+const mgetService = require('../service/mget');
+const logger = require('pelias-logger').get('api');
 
-  function controller( req, res, next ){
+function setup( apiConfig, esclient ){
+  function requestHasErrors(request) {
```

Comment:
I would suggest pulling all the function definitions out of the `setup()` function since they are all independent of the score of that function. The way it is written here they'll be redefined each time the setup function is executed, not to mention keep around the setup function as part of their context, and that's not necessary.